tests/integration/remote/test_polygon.py

```diff
@@ -2,8 +2,10 @@
 # Copyright 2022 Ocean Protocol Foundation
 # SPDX-License-Identifier: Apache-2.0
 #
+import brownie
 import pytest
 from brownie.network import accounts
+from web3.middleware import geth_poa_middleware
```

Comment:
These changes do not belong in this PR. It's a different problem. Create the github issue, create the branch, create the PR, get it approved, merge it. But separately from this one.

Reply:
And, I think the github issue may already exist: https://github.com/oceanprotocol/ocean.py/issues/1187

Reply:
Created #1188